tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -110,6 +112,10 @@ def montecarlo_radial1d(model, plasma, runner):
             np.array(virt_packet_last_line_interaction_out_id)
         ).ravel()
 
+    # Condition for Checking if R Packet Tracking is enabled
+    if montecarlo_configuration.RPACKET_TRACKING:
```

Comment:
We should make sure that if RPACKET_TRACKING is disabled then montecarlo_main_loop should not do any RPacket tracking

Reply:
Yes moved to the discussed location :)